CDC-Data-Reconciliation-Backend/compare.py

```diff
@@ -170,7 +171,7 @@ def main():
 
         writer.writeheader()
         for result in results:
-            writer.writerow({'CaseID': result.caseID, 'EventCode': result.eventCode, 'EventName': result.eventName,'MMWRYear': result.MMWRYear,
+            writer.writerow({'CaseID': result.caseID, 'EventCode': result.eventCode, 'EventName': result.eventName,'MMWRYear': result.MMWRYear, 'CaseClass':result.caseClass,
```

Comment:
Similar change here as mentioned on line 8.

Reply:
I had this already changed, I had just forgot to save the file